app/jobs/calendar_importer/events/ics_event.rb

```diff
@@ -105,7 +104,6 @@ def event_link_types
       {
         'meet.jit.si' =>     { regex: %r{#{http}#{subdomain}meet.jit.si/#{suffix}}, type: 'direct' },
         'meet.google.com' => { regex: %r{#{http}#{subdomain}meet.google.com/#{suffix}}, type: 'direct' },
-        'facebook.com' =>    { regex: %r{#{http}#{subdomain}facebook.com/events/#{suffix}}, type: 'indirect' },
```

Comment:
Missed this on my previous flyover -- I don't think this is part of "old facebook code" either?